styles.css

```diff
@@ -72,7 +72,7 @@ a {
 .multi-filter-container {
   display: flex;
   margin: auto;
-  width: 90%;
+  width: 97%;
```

Comment:
Have we considered other approaches to this yet? We may end up removing the explicit width definition here because the more fragile this styling is, the less flexible it will be to the responsive demands.

Consider viewing the page in mobile views and lets brainstorm some styling that will accommodate for mobile views as well as tablet + desktop.